coin/coin_presentation/src/main/java/com/pavluyk/coin_presentation/viewmodel/CoinDetailedViewModel.kt

```diff
@@ -0,0 +1,30 @@
+package com.pavluyk.coin_presentation.viewmodel
+
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.pavluyk.coin_domain.models.CoinDetailed
+import com.pavluyk.coin_domain.usecases.FetchDetailedDataUseCase
+import kotlinx.coroutines.launch
+
+class CoinDetailedViewModel(
+    private val fetchDetailedDataUseCase: FetchDetailedDataUseCase,
```

Comment:
better name fields with usecases WITHOUT "UseCase" part.
It improves describeability in way:
class - BlaBlaBuisenessLogicUseCase as a store with user intentions -> USE  CASE as is
call - action.execute() (eg. fetchSomeData.execute() )